torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -16,10 +16,13 @@
 
 #include "PyTorchModelLoader.h"
 
+#include "glow/Support/Error.h"
 #include "glow/Support/Support.h"
 
 #include <torch/csrc/jit/ir.h>
 
+using namespace glow;
```

Comment:
You're doing this because the things in Error.h are in `namespace glow` right?
I don't really want to do this because then types like `Tensor` become a bit harder to read if they aren't qualified (is it a PT or a Glow Tensor).
Maybe we could take the things in Error.h out of `namespace glow` or maybe we could just do a couple of `using foo = glow::foo` up top here for the things in Error.h. Thoughts?

Reply:
Yes, the problem with Error.h, which should be fixed first. I will take care of it in another diff, then we can go back to this code and remove "using namespace glow;"

Reply:
Ok sounds good